.github/workflows/schedule-hourly.yml

```diff
@@ -94,3 +94,35 @@ jobs:
           path: |
             bazel-bep.pb
             profile.json
+  bazel-run-fuzzers-hourly:
+    name: Bazel Run Fuzzers Hourly
+    runs-on:
+      labels: dind-large
+    container:
+      image: ghcr.io/dfinity/ic-build@sha256:80e976b63af2b1b352c8c5959cb6c6b02aaa56a4efa327569d8c85c9c81a2cec
+      options: >-
+        -e NODE_NAME --privileged --cgroupns host -v /cache:/cache -v /var/sysimage:/var/sysimage -v /var/tmp:/var/tmp
+    timeout-minutes: 120
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Login to Dockerhub
+        shell: bash
+        run: ./ci/scripts/docker-login.sh
+        env:
+          DOCKER_HUB_USER: ${{ vars.DOCKER_HUB_USER }}
+          DOCKER_HUB_PASSWORD_RO: ${{ secrets.DOCKER_HUB_PASSWORD_RO }}
```

Comment:
is this necessary? do the fuzzer tests need to download the images?

Reply:
I assumed it was a prerequisite for `dind-large-setup`.  But looks like that's not the case and I will remove it.